src/css/root/_mixins.scss

```diff
@@ -168,9 +168,20 @@
 
 // Clearfix
 @mixin clearfix {
-	&::after {
-	  clear: both;
-	  content: "";
-	  display: table;
+	*zoom: 1;
```

Comment:
le zoom c'est old @Nkzq look : https://css-tricks.com/clearfix-a-lesson-in-web-development-evolution/#article-header-id-3

Reply:
@Cocola j'avais lu un article justement disant que c'était l'implémentation la plus safe en matière de browser support.

Reply:
ah la y a pas de doute, ie6/7 ready 🤣  mais BFF c'est ie9+ voir 10 ^^

Reply:
En soit ça coûte rien de l'enlever ou le laisser